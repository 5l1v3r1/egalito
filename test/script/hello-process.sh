#!/bin/bash
mkdir -p tmp

ln -s ../../src/libegalito.so
LD_LIBRARY_PATH=../../src EGALITO_DEBUG=/dev/null ../../src/loader \
	../binary/build/fork \
    >tmp/fork.out 2>&1
rm libegalito.so

if [ -z "$(diff hello-process.expected tmp/fork.out)" ]; then
    echo "test passed"
else
    echo "test failed!"
    exit 1
fi
