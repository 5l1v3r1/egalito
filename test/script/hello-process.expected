[parent] Hello, World!
[child] Hello, World!
